packages/react-integration/demo-app-ts/src/components/demos/NavDemo/NavDemo.tsx

```diff
@@ -59,7 +59,7 @@ export class NavDemo extends Component {
         <Title headingLevel="h2" size="2xl">
           Default Nav
         </Title>
-        <div className="example" style={{ border: '1px solid rgb(114, 118, 123)' }}>
+        <div className="example" style={{ backgroundColor: '#212427', border: '1px solid rgb(114, 118, 123)' }}>
```

Comment:
why do we need this change?

Reply:
@tlabaj the nav demo looked like this before adding that styling: https://github.com/patternfly/patternfly/issues/3279. per @redallen, that is actually our default nav styling! He mentioned it needs to be on a dark background for it to look right.